PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!-- Describe what has changed in this PR -->
```

Comment:
Thoughts on keeping this? We could evolve this and has been mentioned that we need one.